ruby_basic/nikolay.braslavets/home_task_1/8.rb

```diff
@@ -0,0 +1,27 @@
+=begin
+#This variant if you need to set the dimension of the array
+
+puts "Hi enter the words/numbers of which we will make an array."
+
+array = Array.new
+
+10.times do
+  str = gets.chomp
+  array.push str
+  array.sort.reverse
+end
+puts "Sotring result:"
+print array
+=end
+
+puts 'Input the words/numbers to make an array. Enter twice to stop :)'
+
+array = [ ]
```

Comment:
Do not use spaces when defining empty array. Use [] instead.

Reply:
This is an accident, fix.